libs/feature/editor/src/lib/components/record-form/form-field/form-field-keywords-generic/form-field-keywords-generic.component.ts

```diff
@@ -0,0 +1,80 @@
+import { CommonModule } from '@angular/common'
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+} from '@angular/core'
+import { FormControl } from '@angular/forms'
+import { PlatformServiceInterface } from '@geonetwork-ui/common/domain/platform.service.interface'
+import {
+  AutocompleteComponent,
+  DropdownSelectorComponent,
+  UiInputsModule,
+} from '@geonetwork-ui/ui/inputs'
+import { UiWidgetsModule } from '@geonetwork-ui/ui/widgets'
+import { map } from 'rxjs'
+import { Keyword } from '@geonetwork-ui/common/domain/model/record'
+import { KeywordType } from '@geonetwork-ui/common/domain/model/thesaurus'
+
+type AutocompleteItem = { title: string; value: Keyword }
+
+@Component({
+  selector: 'gn-ui-form-field-keywords-generic',
```

Comment:
Change name to `gn-ui-form-field-generic-keywords` as the name of this component. 
Are there better ideas for naming?

Reply:
I'm not really sure what this component is responsible of, as the keywords have already been dealt with in an another already merged PR. If this is the common part between the two features, this should be removed from the other form-field (didn't see any dropped code in this PR). And you could place it in feature/editor/src/lib/components, and the drop the "form-field" part of the name.